libmamba/src/core/prefix_data.cpp

```diff
@@ -231,7 +233,15 @@ namespace mamba
         );
         if (ec)
         {
-            throw std::runtime_error(ec.message());
+            const auto message = fmt::format(
```

Comment:
This is good to have for more details in general indeed.
But in this specific issue, I think the problem is parsing the [output](https://github.com/mamba-org/mamba/blob/main/libmamba/src/core/prefix_data.cpp#L244) which is not valid so maybe add a debug log or even a trace log (to not disturb other debugging use cases too much, if we think the log should stay permanently):
```
LOG_TRACE << "Parsing `pip inspect` output:\n" << out;
nlohmann::json j = nlohmann::json::parse(out);
```

Reply:
Ah yes wrong try/catch, thanks I'll move it.

Reply:
@Hind-M I suspect we might want these info every time there is a parsing error there without having to add trace level of logging. Or do you think it would be too much and the trace is enough?

Reply:
Ok I'll attempt with a mix of trace and throw and lets see how it looks concretely.